test/audits/html/theme-colors.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const Audit = require('../../../audits/html/theme-color.js');
+const assert = require('assert');
+
+/* global describe, it*/
+
+describe('HTML: theme-color audit', () => {
+  it('fails when no html present', () => {
+    return assert.equal(Audit.audit({}).value, false);
+  });
+
+  it('fails when no theme-color present in the html', () => {
+    return assert.equal(Audit.audit({
+      html: `<!doctype html>
+              <html>
+                <head>
+                  <title>Sample page</title>
+                </head>
+                <body></body>
+              </html>`
+    }).value, false);
+  });
+
+  it('fails when theme-color has no content value', () => {
+    return assert.equal(Audit.audit({
+      html: `<!doctype html>
+              <html>
+                <head>
+                  <meta name="theme-color" content="">
+                  <title>Sample page</title>
+                </head>
+                <body></body>
+              </html>`
+    }).value, false);
+  });
+
+  it('fails when multiple theme-colors exist', () => {
+    return assert.equal(Audit.audit({
+      html: `<!doctype html>
```

Comment:
can we DRY up these html mocks with a single function like

getMockHtml: function (insert) {
  return `...
   ...
  ${insert}
   ....
   ....`
}